src/parser/jobs/drg/modules/Procs.tsx

```diff
@@ -0,0 +1,75 @@
+
+export default class Procs extends CoreProcs {
+	trackedProcs = [
+		{
+			procStatus: this.data.statuses.SHARPER_FANG_AND_CLAW,
+			consumeActions: [this.data.actions.FANG_AND_CLAW],
+		},
+		{
+			procStatus: this.data.statuses.ENHANCED_WHEELING_THRUST,
+			consumeActions: [this.data.actions.WHEELING_THRUST],
+		},
+		{
+			procStatus: this.data.statuses.DIVE_READY,
+			consumeActions: [this.data.actions.MIRAGE_DIVE],
+		},
+	]
+
+	protected addJobSpecificSuggestions(): void {
+		const droppedFang = this.getDropCountForStatus(this.data.statuses.SHARPER_FANG_AND_CLAW.id)
+		const droppedWheeling = this.getDropCountForStatus(this.data.statuses.ENHANCED_WHEELING_THRUST.id)
+		const droppedMirage = this.getDropCountForStatus(this.data.statuses.DIVE_READY.id)
+		const overwrittenDiveReady = this.getOverwriteCountForStatus(this.data.statuses.DIVE_READY.id)
+
+		this.suggestions.add(new TieredSuggestion({
+			icon: droppedFang >= droppedWheeling ? this.data.actions.FANG_AND_CLAW.icon : this.data.actions.WHEELING_THRUST.icon,
+			content: <Trans id="drg.procs.suggestions.extenders.content">
+				Avoid interrupting your combos at the <ActionLink {...this.data.actions.FANG_AND_CLAW}/> and <ActionLink {...this.data.actions.WHEELING_THRUST}/> stages, as it causes you to lose the procs that allow you to cast them, costing you both the cast and the <ActionLink {...this.data.actions.BLOOD_OF_THE_DRAGON}/> duration that comes with it.
+			</Trans>,
+			tiers: {
+				1: SEVERITY.MINOR,
+				2: SEVERITY.MEDIUM,
+				4: SEVERITY.MAJOR,
+			},
+			value: droppedFang + droppedWheeling,
+			why: <Trans id="drg.procs.suggestions.extenders.why">
+				You dropped <Plural value={droppedFang} one="# Fang and Claw proc" other="# Fang and Claw procs"/> and <Plural value={droppedWheeling} one="# Wheeling Thrust proc" other="# Wheeling Thrust procs"/>.
+			</Trans>,
+		}))
+
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.actions.MIRAGE_DIVE.icon,
+			content: <Trans id="drg.procs.suggestions.mirage-dropped.content">
+				Avoid letting your <StatusLink {...this.data.statuses.DIVE_READY}/> procs fall off, as it can delay your Life of the Dragon windows and potentially cost you a lot of DPS.
+			</Trans>,
+			tiers: {
+				1: SEVERITY.MEDIUM,
+				3: SEVERITY.MAJOR,
+			},
+			value: droppedMirage,
+			why: <Trans id="drg.procs.suggestions.mirage-dropped.why">
+				You dropped <Plural value={droppedMirage} one="# Mirage Dive proc" other="# Mirage Dive procs"/>.
+			</Trans>,
+		}))
+
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.actions.MIRAGE_DIVE.icon,
+			content: <Trans id="drg.procs.suggestions.mirage-overwritten.content">
+				Avoid casting <ActionLink {...this.data.actions.JUMP}/> and <ActionLink {...this.data.actions.SPINESHATTER_DIVE}/> when you already have a <StatusLink {...this.data.statuses.DIVE_READY}/> procs, as it overwrites them and can delay your Life of the Dragon windows and potentially cost you a lot of DPS.
```

Comment:
```suggestion
				Avoid casting <ActionLink {...this.data.actions.HIGH_JUMP}/> when you already have a <StatusLink {...this.data.statuses.DIVE_READY}/> procs, as it overwrites them and can delay your Life of the Dragon windows and potentially cost you a lot of DPS.
```

Per discussion in #dev-melee - looks like this never got updated for ShB, might as well finally fix it while we're touching it lol

Reply:
It's dead, Jim.